docs/tutorials/writing_docs.rst

```diff
@@ -0,0 +1,244 @@
+Writing software documentation
+==============================
+
+This page is meant to serve both as a tutorial for writing documentation and
+as example of what that documentation should look like. For example, each
+page should start with a top-level header, an introductory paragraph or two, and
+a table of contents.
+
+.. contents::
+
+Overview
+--------
+
+The overview should tell the reader how all the pieces fit together.
+In this case our goal is to give the user documentation that is
+**consistent**, **concise**, and **demonstrative**. This is a good place for
+diagrams to demonstrate how this part relates to the rest of the system, or
+how the parts of this document relate to one-another.
+
+.. figure:: ../imgs/stupid_diagram.png
+   :scale: 100%
+   :alt: This diagram is a silly example.
+
+   This diagram's purpose is to demonstrate the use of diagrams, and to show
+   that captions should be used to explain each diagram's purpose. This was
+   generated using the 'Drawing' app in Google Docs.
+
+Requirements
+------------
+
+
+For tutorials like this one, you generally want to start with a list of
+requirements so that the reader can get together everything they need to
+follow along. It might include things like links to
+:ref:`installing-pavilion`, and a bulleted list of requirements:
+
+ - Basic familiarity with Restructured Text and Sphinx
+
+   - Sphinx has nice `Sphinxified ReST documentation`_.
+ - Also see ``docs/README.md`` for instructions on building and debugging the
+   doc tree.
+ - A copy of the source of this document.
+
+   - While this isn't a lesson on *how* to use Sphinx/ReST, I do try to
+     demonstrate in this document as many features as I can.
+ - *(optional)* You may want to include optional requirements that will
+   make life easier for the reader.
+
+For non-tutorial documentation, you can assume by that point the user has the
+system set up.
+
+.. _Sphinxified ReST documentation:  http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
+
+Concise
+-------
+
+Users should be able to find the information they need in the documentation
+as quickly as possible. :ref:`Consistent` style and structure along with
+:ref:`demonstrative` examples are the cornerstones of concise documentation
```

Comment:
This reads as "along with Being Demonstrative examples" in the html. I think it should show as "along with Demonstrative examples" instead.

Reply:
I'm still getting the hang of this style of link.